pnpm/test/server.ts

```diff
@@ -177,17 +174,19 @@ skipOnWindows('installation using store server started in the background', async
   await execPnpm(['install', 'is-positive@1.0.0', '--use-store-server'])
 
   const serverJsonPath = path.resolve('..', 'store/v3/server/server.json')
-  const serverJson = await retryLoadJsonFile<{ connectionOptions: object }>(serverJsonPath)
-  expect(serverJson).toBeTruthy()
-  expect(serverJson.connectionOptions).toBeTruthy()
 
-  expect(project.requireModule('is-positive')).toBeTruthy()
-
-  await execPnpm(['uninstall', 'is-positive'])
+  try {
```

Comment:
This test never calls `pnpm server start`, so I opted to just use a `try / finally` block.